docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc

```diff
@@ -1,10 +1,9 @@
-[[_fine_grain_permissions]]
 
-=== Fine grain admin permissions
+=== Fine grained admin permissions V1
 
-:tech_feature_name: Fine Grain Admin Permissions
-:tech_feature_id: admin-fine-grained-authz
-include::../templates/techpreview.adoc[]
+IMPORTANT: Fine-grained Admin Permissions V1 have been replaced by a <<_fine_grain_permissions, new version>>. 
+Version 1 of the feature is still marked as preview and is available, but it may be deprecated and removed 
+in future. To enable it start the server with `--features=admin-fine-grained-authz:v1`.
```

Comment:
```suggestion
IMPORTANT: Fine-grained Admin Permissions V1 have been replaced by a <<_fine_grain_permissions, new version>>. 
Version 1 of the feature is still marked as preview and is available, but it may be deprecated and removed 
in future. To enable it, start the server with `--features=admin-fine-grained-authz:v1`.
```